jdbc-sink/src/test/java/org/springframework/cloud/stream/module/jdbc/ShorthandMapConverterTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.jdbc;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.collection.IsMapContaining.*;
+import static org.junit.Assert.*;
```

Comment:
I think you are correct here: https://github.com/spring-projects/spring-framework/blob/master/src/idea/spring-framework.xml#L16.

And SF team really uses asterisk imports: 
https://github.com/spring-projects/spring-framework/blob/master/spring-websocket/src/test/java/org/springframework/web/socket/client/WebSocketConnectionManagerTests.java#L37
 
https://github.com/spring-projects/spring-framework/blob/master/spring-jms/src/test/java/org/springframework/jms/annotation/EnableJmsTests.java#L47

But should we do the same in our projects?..